pkg/operator/resourcesynccontroller/resourcesynccontroller.go

```diff
@@ -50,6 +50,12 @@ func NewResourceSyncController(
 	); err != nil {
 		return nil, nil, err
 	}
+	if err := resourceSyncController.SyncSecret(
+		resourcesynccontroller.ResourceLocation{Namespace: operatorclient.TargetNamespace, Name: "encryption-config"},
```

Comment:
won't it be synced in `NewRevisionLabelPodDeployer?

Reply:
yep. Will remove thi sone.